extensions/intellij/intellij_generator_plugin/src/test/resources/generator/RxTestBlocGoldenGenerator/RxTestBlocAlchemistGoldenGenerator_all.dart

```diff
@@ -0,0 +1,25 @@
+import '../../helpers/golden_helper.dart';
+import '../factory/sample_factory.dart';
+
+
+void main() {
+  runGoldenTests([
+    buildScenario(
+      scenario:  'sample_empty',
+      widget: sampleFactory(),
+    ),
+    buildScenario(
+      scenario:  'sample_success',
+      widget: sampleFactory(),
+    ),
+    buildScenario(
+      scenario:  'sample_loading',
+      customPumpBeforeTest: animationCustomPump,
```

Comment:
It looks like the `animationCustomPump` does not solve the animation issues. @DDavidPrime  / @GoranPrime please let us know when you find a stable solution so that @PVGeorgievPrime can integrate it in the plugin itself.

Reply:
Yes, this is no more the case, so it can be safely removed.

Reply:
Removed